packages/angular/src/utils/backward-compatible-versions.ts

```diff
@@ -33,7 +33,6 @@ export const backwardCompatibleVersions: Record<
     autoprefixerVersion: '^10.4.0',
     tsNodeVersion: '10.9.1',
     jestPresetAngularVersion: '~12.2.3',
-    protractorVersion: '~7.0.0',
```

Comment:
Please check other package versions here that were only needed for Protractor and remove them as well.